srcdoc/configure/access_control.mt

```diff
@@ -0,0 +1,243 @@
+? my $ctx = $main::context;
+? $_mt->wrapper_file("wrapper.mt", "Configure", "Access Control")->(sub {
+
+<p>
+Starting from version 2.1, H2O comes with a DSL-like mruby library which make it easy to write access control list (ACL).
+</p>
+
+<h2 id="example" class="section-head">Example</h2>
+
+<p>
+Below example uses this Access Control feature to write various access control.
+</p>
+
+<?= $ctx->{example}->('Access Control', <<'EOT');
+paths:
+  "/":
+    mruby.handler: |
+      acl {
+        allow { addr == "127.0.0.1" }
+        deny { user_agent.match(/curl/i) && ! addr.start_with?("192.168.") }
+        respond(503, {}, ["Service Unavailable"]) { addr == malicious_ip }
+        redirect("https://somewhere.com/", 301) { path =~ /moved/ }
+        use Htpasswd.new("/path/to/.htpasswd", "realm") { path.start_with?("/admin") }
+      }
+    file.dir: /path/to/doc_root
+EOT
+?>
+
+<p>
+In the example above, the handler you got by calling <code>acl</code> method will do the following:
+<ul>
+  <li>
+    If the remote IP address exactly equals to "127.0.0.1", the request will be delegated to the next handler (i.e. serve files under /path/to/doc_root) and all following acl settings is ignored.
+  </li>
+  <li>
+    Otherwise, if the user agent string includes "curl" and the remote IP address doesn't start with "192.168.", this handler immediately returns <code>403 Forbidden</code> response.
+  </li>
+  <li>
+    Otherwise, if the remote IP address is exactly equals to the <code>malicious_ip</code> variable, this handler immediately returns <code>503 Service Unavailable</code> response.
+  </li>
+  <li>
+    Otherwise, if the request path matches with the pattern <code>/moved/i</code>, this handler immediately redirects the client to <code>"https://somewhere.com"</code> with <code>301</code> status code.
```

Comment:
It might be polite to use one of the reserved domain names for examples (i.e. those defined in https://tools.ietf.org/html/rfc2606 and other docs).